Dockerfile

```diff
@@ -1,6 +1,5 @@
 FROM navikt/node-express:14-alpine
 USER root
-RUN apk --no-cache add curl
```

Comment:
Hva var det denne gjorde?

Reply:
apk er Alpine sitt verktyg for å laste ned pakkar (ca tilsvarande apt på debian, trur eg), så denne linja lasta ned curl.

Men korfor vi skulle trengje curl inne i containeren? Godt spørsmål. Det eg fann frå gitloggen er at denne har vore med heilt sia initialiseringa av prosjektet.

Kaanskje @hensol eller @kristeine veit?